Translation Script/Enemies.csv

```diff
@@ -1,7 +1,7 @@
 "テスト用雑魚","Minions for Testing"
 "ルナ","Luna"
-"2戦目","2Battle Eye"
-"\n[1]",
+"2戦目","Battle Eye"
+"\n[1]"," can you see anything we can use?"""
```

Comment:
This line doesn't look quite right. Was this intentional?

Reply:
Yea I think I made a mistake randomly, my bad